central/processbaseline/service/service_impl.go

```diff
@@ -72,14 +77,41 @@ func (s *serviceImpl) GetProcessBaseline(ctx context.Context, request *v1.GetPro
 	}
 	baseline, exists, err := s.dataStore.GetProcessBaseline(ctx, request.GetKey())
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrapf(errox.NotFound, "No process baseline with key %+v found", request.GetKey())
```

Comment:
This changed the existing endpoint. Users need to get the point that process baseline does not always exist.